### ISSUE TITLE
Tachikiri Fever

### ISSUE THREAD

**Issue by Boy0000** [MineInAbyss/Issue-Tracker#72]
### Is your feature request related to a problem?

Layer 5 exclusive mobs are very non-problematic.

### Describe the solution you'd like.

_Tachikiri Fever_ is a canon disease delvers get in layer 2 from a mob with the same name. We currently have an "evolved" version of it, namely **Kagi Tachikiris** in Layer 5 which could have a % to infect the player with it rather than just poison.

Currently very limited on custom Status Effects to simple renames of vanilla, but the Idea is there.

### Describe alternatives you've considered.

Well keep it boring poison.

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported Paper build.

### Other
Would also be able to craft or buy an antidote for this using a custom brewing stand recipe perhaps

[Original Wiki article](https://madeinabyss.fandom.com/wiki/Tachikiri_Fever)

**Comment by VaIienta** [comment]
Yeah I remember we spoke about this with design team about having there be an antidote cure for it that you would purchase if your going to be delving in L2, and it would be a nice addition to the curse for L2 but considering that the curse in L2 is already poison might need to wait until we can do something more interesting with the fever
